src/instrumentation_measurement/measurements.py

```diff
@@ -66,6 +78,17 @@ def run_geth(self, program, sampleSize):
 
     return instrumenter_result
 
+  def run_evmone(self, program, sampleSize):
+    evmone_build_path = './instrumentation_measurement/evmone/build/'
+    evmone_main = [evmone_build_path + 'evmc/bin/evmc', 'run']
+    args = ['--vm', evmone_build_path + '/lib/libevmone.so', '--measure-all', '--repeat', '{}'.format(sampleSize)]
+    invocation = evmone_main + args + [program.bytecode]
+    result = subprocess.run(invocation, stdout=subprocess.PIPE, universal_newlines=True)
+    assert result.returncode == 0
+    instrumenter_result = result.stdout.split('\n')[2:-5]
```

Comment:
pls leave a comment what the 2 and -5 are here. (also you can add a comment for geth above, which I forgot: `# -1 is to strip the final \n`)